src/itpp.mk

```diff
@@ -1,7 +1,7 @@
 # This file is part of MXE. See LICENSE.md for licensing information.
 
 PKG             := itpp
-$(PKG)_WEBSITE  := http://itpp.sourceforge.net
+$(PKG)_WEBSITE  := https://itpp.sourceforge.io/
```

Comment:
Just redirects the http://itpp.sourceforge.net site

Reply:
I know, it needs to be fixed by the project by enabling HTTPS on the SourceForge admin panel. I've made this change for all other sf homepages, so I think this should pass too.